scripts/gen_i18n.py

```
@@ -684,20 +703,43 @@ def _print_language_table(
+        "Own",
+        "Fallback",
+        "Unused",
+        "Data (B)",
+        "Unique (B)",
+    )
 
     rows = []
-    for code, name, inherited, size in zip(
-        language_codes, language_names, inherited_sets, data_sizes
+    for lang_idx, (code, name, inherited, size) in enumerate(
+        zip(language_codes, language_names, inherited_sets, data_sizes)
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)